core/src/test/java/com/hcl/ecomm/core/servlets/MyAccountAddAddressTest.java

```diff
@@ -0,0 +1,125 @@
+package com.hcl.ecomm.core.servlets;
```

Comment:
@ShubhamAMU : pl remane to MyAccountAddAddressServletTest

Reply:
@rishabh-behl  thats already done. Please check the files.